app/NewOrder.php

```diff
@@ -273,8 +279,10 @@ public function toArray()
             "late_fee_gen" => $this->late_fee_gen,
             "id" => $this->id,
             "order_number" => $this->order_number,
+            "bnpl_vendor_product_id" => $this->bnpl_vendor_product_id,
```

Comment:
Hey @harmlessprince i was hoping we'd use polymorphic relationship to handle this.